### PR TITLE
EOS-13069: sspl files path change due to disable pyinstaller

### PR DESCRIPTION
**Pull request by satish-seagate**

### CODE REVIEW

**Review by ypise — APPROVED** [review]

**Comment by satish-seagate** [comment]
Need approval from @cdeshmukh after that only we can go ahead for merge. SSPL code was not merged yet which needs parallel goes with this change. (https://github.com/Seagate/cortx-monitor/pull/259)

FYI - @ypise @83bhp

**Pull request merged**
